fastlane/Fastfile

```diff
@@ -94,6 +125,42 @@ platform :android do
     tag_release_with_latest_if_needed(release_version: release_version)
   end
 
+  desc "Make github release"
+  lane :github_amazon_release do |options|
+    release_version = options[:version]
+    fail ArgumentError, "missing version" unless release_version
+
+    begin
+      changelog = File.read("../CHANGELOG.latest.md")
+    rescue
+      fail "please add a CHANGELOG.latest.md file before calling this lane"
+    end
+
+    add_git_tag(
+      tag: "#{release_version}",
+      force: true,
+      grouping: "releases"
+    )
+
+    push_git_tags(
+      tag: "#{release_version}",
+      force: true
+    )
+
+    set_github_release(
+      repository_name: "revenuecat/purchases-android",
+      api_token: ENV["GITHUB_TOKEN"],
+      name: release_version,
+      tag_name: "#{release_version}",
+      description: changelog,
+      commitish: "main",
+      upload_assets: [],
+      is_prerelease: true
```

Comment:
👍

Reply:
so should we be setting the changelog to the amazon instructions when we release?

Reply:
Good idea!